template/.npmignore.temp

```diff
@@ -12,11 +14,15 @@ docs/
 scripts/
 test/
 apiExamples/
+docTemplates/
```

Comment:
I think this can also get added to the `.gitignore` since this is fetched from the server when we build the docs.